client/src/Components/NavigationBar.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Box from "@material-ui/core/Box";
+import Button from '@material-ui/core/Button';
+import Typography from "@material-ui/core/Typography";
+import DashboardRoundedIcon from '@material-ui/icons/DashboardRounded';
+import CalendarTodayOutlinedIcon from '@material-ui/icons/CalendarTodayOutlined';
+import AddIcon from '@material-ui/icons/Add';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import IconButton from '@material-ui/core/IconButton';
+
+const useStyles = makeStyles(theme => ({
+    appBar: {
```

Comment:
the height of the app bar need to be little larger to match the wireframe

Reply:
Implemented.